src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -128,8 +129,7 @@ await using (var stream = await BackupUtils.GetDecompressionStreamAsync(new Buff
                         using (var source = new StreamSource(stream, context, Database.Name))
                         {
                             var destination = Database.Smuggler.CreateDestination();
-
-                            var smuggler = Database.Smuggler.Create(source, destination, context);
+                            var smuggler = Database.Smuggler.Create(source, destination, context, new DatabaseSmugglerOptionsServerSide(GetAuthorizationStatus(HttpContext.Features.Get<IHttpAuthenticationFeature>() as AuthenticateConnection)));
```

Comment:
Add `GetAuthorizationStatusFromHttpContext`.
There is a lot of usage of `HttpContext.Features.Get<IHttpAuthenticationFeature>() as AuthenticateConnection)`